tests/unit/conditions/test_condition_validity.py

```diff
@@ -111,3 +111,17 @@ def test_invalid_contract_condition():
                 ":hrac",
             ],
         )
+
+    # standard contract type and function ABI
+    with pytest.raises(InvalidCondition):
+        _ = ContractCondition(
+            contract_address="0xaDD9D957170dF6F33982001E4c22eCCdd5539118",
+            method="getPolicy",
```

Comment:
Let's specify an actual ERC20 method here. One of the failure cases is the method not being valid for the ERC20 contract type, which `getPolicy` is not. I'm not saying that's why this test is failing (I believe it is checked after checking whether both function_abi and standard contract type are set), but it is a failure case.